tasks/relationships.js

```diff
@@ -1,41 +1,41 @@
-const getRelationshipsCollection = require('../config/mongoCollections').relationships;

```

Comment:
I guess you haven't updated this file at all. 
You can reset or checkout and commit in your next commit. Please feel free to reach out to Danni or me if you face any issue to do that.